aichat/openapi/aichat.json/openapi.yaml

```diff
@@ -0,0 +1,88 @@
+openapi: 3.0.0
+info:
+  title: Chat Protocol
+  version: 0.0.0
+tags: []
+paths:
+  /chat:
+    post:
+      operationId: getStreamedCompletion
+      parameters: []
+      responses:
+        '200':
+          description: The request has succeeded.
+          content:
+            application/jsonl:
+              schema:
+                type: string
```

Comment:
I am a little confused by this, does application/jsonl necessarily mean streaming?

Reply:
I feel like there's some information loss between typespec and oas3 output.

Reply:
This is a fallback schema for open api 3 output since open api 3 doesn't have a standard way to describe streaming operations. This still allows openapi codegen to do _something_ with this output.

For TypeSpec-based code generators, they would get the stream metadata from the http operation and from that know how to treat the stream.